animations/linear_regression/lr2_anim.py

```diff
@@ -0,0 +1,107 @@
+import numpy as np
```

Comment:
Is this the original animation file before I did the rewrite (e.g., adding the util classes and other animations)? I think you can delete this if it's stale.

Reply:
sorry yeah lr_anim.py and lr2_anim.py are just placeholders, we can just delete them and integrate your actual changed in another pr